docker/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Run tests and make jar
+# Uses docker.
+
+set -eux
+
+# check if mysql is running
+mysql_running=$(docker inspect -f {{.State.Running}} mysql)
+if ! $mysql_running ; then
+    # start up mysql
+    docker pull mysql/mysql-server:5.7.15
+    docker run --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d -p 3310:3306 mysql/mysql-server:5.7.15
+
+    # validate mysql
+    sudo yum -y install mysql
```

Comment:
Would this mean we no longer have a way to build docker images locally?  What would be the best way for devs to build images for local/experimental code?